e2e/cypress/integration/messaging/center_channel_rhs_overlap_spec.js

```diff
@@ -49,5 +81,810 @@ describe('Messaging', () => {
 
         // * Check if "Add Comment" button is visible
         cy.get('#addCommentButton').scrollIntoView().should('be.visible').and('have.value', 'Add Comment');
+
+        // # Reset the viewport
+        cy.viewport(1280, 900);
+    });
+
+    it('MM-T712_1 Editing a post and hitting <enter> in code block saves post automatically instead of adding newline', () => {
```

Comment:
I edited/reduced the original scope of [MM-T712](https://mattermost.atlassian.net/projects/MM?selectedItem=com.atlassian.plugins.atlassian-connect-plugin%3Acom.kanoah.test-manager__main-project-page#!/testCase/MM-T712) to match with the test included in this `it` block.
```suggestion
    it('MM-T712 Editing a post with Ctrl+Enter on for all messages configured', () => {
```